app/models/plate_purpose/broadcast_library_complete.rb

```diff
@@ -1,25 +0,0 @@
-module PlatePurpose::BroadcastLibraryComplete # rubocop:todo Style/Documentation
```

Comment:
We have a number of these modules which were confined to a single purpose class, in some cases one which has been deprecated.

I've inlined all modules which only belonged to a single class.